extension/src/help/openWebHelp.ts

```diff
@@ -0,0 +1,212 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+export let webHelpContainer: WebHelpLibrary;
+
+//#region Web Help Initialization
+// Invoked from extension.ts to setup the Right Click 'Open Online Help' command. 
+export function registerHelpCommands(context: vscode.ExtensionContext) {
+	context.subscriptions.push(vscode.commands.registerCommand('autolisp.openWebHelp', async () => {
+		try {
+			await openWebHelp();
+		}
+		catch (err) {
+			if (err) {
+				console.log(err.toString());
+			}
+		}
+	}));
+}
+
+// Invoked from extension.ts as a generic early-loader of the webHelpAbstraction; possibly future related files too.
+export function readAllHelpData() {
+	loadWebHelpAbstraction("../../extension/src/help/webHelpAbstraction.json");	
+}
+
+function loadWebHelpAbstraction(datafile: string): void {
+	webHelpContainer = new WebHelpLibrary();
+    var fs = require("fs");
+    var dataPath = path.resolve(__dirname, datafile);
+    fs.readFile(dataPath, function(err, data) {        
+        if (err === null) {                      
+            webHelpContainer.load(JSON.parse(data));
+        }
+    });    
+}
+//#endregion
+
+
+// Triggers VSCode to open the official web documentation URL related to the currently selected symbol
+// Note: this function is directly referenced by projectCommands.ts and the the package.json contributes (commands & menus) section.
+export async function openWebHelp() {
+	const editor: vscode.TextEditor = vscode.window.activeTextEditor;
+	let selected: string = editor.document.getText(editor.selection);	
+	if (selected === "") {
+		await vscode.commands.executeCommand('editor.action.addSelectionToNextFindMatch');
```

Comment:
This selects word under the cursor. It seems to work for me.

Reply:
I had this exact same problem where *editor.action.addSelectionToNextFindMatch* wasn't actually selecting text until after my function had fully completed. After an updated call to *./winCompile.bat* it suddenly started working. 

I also understand your confusion because the name implies we already have a selection. If you search the page for the word "tribal" you'll find my comments on this with szilvaa-adsk.

Reply:
OK... I guess it's the cleanup step in the./winCompile.bat that helped. This batch file is supposed to do a clean build.